src/middleware/verifyToken.js

```diff
@@ -0,0 +1,14 @@
+import jwt from 'jsonwebtoken';
+
+// eslint-disable-next-line consistent-return, func-names
+export default function (req, res, next) {
```

Comment:
Adding rules to eslint means they apply globally. I need to enforce the consistent return through out the application except for very specific cases. This means using these comments in those specific cases. Otherwise swtiching these off globally makes the use of esling redundant.